index_rs/src/index_rs/tests/tests.sh

```diff
@@ -17,6 +17,10 @@ call IndexCanister.register_client(record
           { client_canister_id = client_canister_id });
 assert _ != (0 : nat);
 
+// should successfully query the current client
+call IndexCanister.current_client();
+assert _ != (null : opt null);
```

Comment:
nit: we could be a bit more specific as follows: 
```suggestion
assert _ == opt record { client_canister_id = client_canister_id };
```

Reply:
done